applications/auto_setup/backend/setup_cavity.py

```diff
@@ -36,9 +36,30 @@ def __init__(
         self.note_pv: str = self.auto_pv_addr("NOTE")
         self._note_pv_obj: Optional[PV] = None
 
+        if self.number is not None:
+            prefix = "RFS1" if cavity_num % 4 in (1, 2) else "RFS2"
```

Comment:
Also now that I'm looking at this, can you add tests to make sure that the cavity prefixes are mapping correctly? Like, if is cavity 1 assert prefix is RFS1A etc